st2common/st2common/policies/__init__.py

```diff
@@ -0,0 +1,16 @@
+# Licensed to the StackStorm, Inc ('StackStorm') under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from st2common.policies.base import *   # noqa
```

Comment:
Minor style thing - that's a bad practice which we should avoid doing in the new code. It's especially bad if you don't define `__all__` since it breaks a lot of "static" analysis (e.g. pylint) and other things. And even if you define `__all__` it's error prone since it can get out of date.

The best thing to do is to import directly from the base module. In most cases `__init__.py` should just be empty.